### ISSUE TITLE
bug: Compilation fails on FreeBSD 13.0. Undeclared IP_PKTINFO

### ISSUE THREAD

**Issue by dmilith** [nghttp2/nghttp2#1620]
```
util.cc:1695:64: error: use of undeclared identifier 'IP_PKTINFO'
      if (cmsg->cmsg_level == IPPROTO_IP && cmsg->cmsg_type == IP_PKTINFO) {
                                                               ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
2 errors generated.
```

**Comment by tatsuhiro-t** [comment]
Thanks.  I'll guard this function under ENABLE_HTTP3 macro.

**Comment by tatsuhiro-t** [comment]
Fix committed via 318e0c8447e2399e7c1e95d8ae8d00118ccea5f7
